.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ on:
       - 'v[0-9]+.[0-9]+.[0-9]+' #semantic version format
 
   schedule:
-    - cron: '01 07 * * *'
+    - cron: '00 07 * * 5'
```

Comment:
This can be whatever time you want of course. And doing it weekly is not unreasonable. I wonder if we should change that in the addon-template.

Reply:
You are right that the weekly schedule is not good, but I have another idea that came to my mind, when ddev gets a newer version the addon triggers its own workflow, for example for a stable or nightly build the addon checks if it should trigger a workflow and check if the current version of the addon is compatible with the newer version, I found something like this on the marketplace action has parameters for the owner and repo and according to which workflow should follow but will it work on github in the current form it probably needs to be tested and additionally cron parameters if someone wants to have frequent checking workflow. 

https://github.com/convictional/trigger-workflow-and-wait

Reply:
Well, so many things can go wrong, it's important to check regularly. I still like the regular checking, whether nightly or weekly.